packages/ibm-products/src/components/WebTerminal/WebTerminal.tsx

```diff
@@ -176,7 +176,7 @@ export let WebTerminal = React.forwardRef(
         }}
         onAnimationEnd={onAnimationEnd}
       >
-        <header className={`${blockClass}__bar`}>
+        <header aria-label="Web terminal bar" className={`${blockClass}__bar`}>
```

Comment:
We shouldn’t hardcode this so we may need to add a new prop for this since it doesn’t look like we have any other label this could reference.